### ISSUE TITLE
Crash when the server responded the wrong format

### ISSUE THREAD

**Issue by chalkpe** [marzn/telegrambot-japi#2]
```
03:39:43.092 [Thread-1] DEBUG de.vivistra.telegrambot.client.Bot - POST https://api.telegram.org/******/getUpdates HTTP/1.1
03:40:03.355 [Thread-1] DEBUG de.vivistra.telegrambot.client.Bot - HTTP/1.1 200 OK
03:40:03.355 [Thread-1] DEBUG de.vivistra.telegrambot.client.Bot - {"ok":true,"result":[]}
03:40:03.355 [Thread-1] DEBUG de.vivistra.telegrambot.client.Bot - POST https://api.telegram.org/******/getUpdates HTTP/1.1
03:40:23.619 [Thread-1] DEBUG de.vivistra.telegrambot.client.Bot - HTTP/1.1 200 OK
03:40:23.619 [Thread-1] DEBUG de.vivistra.telegrambot.client.Bot - {"ok":true,"result":[]}
03:40:23.619 [Thread-1] DEBUG de.vivistra.telegrambot.client.Bot - POST https://api.telegram.org/******/getUpdates HTTP/1.1
03:40:43.882 [Thread-1] DEBUG de.vivistra.telegrambot.client.Bot - HTTP/1.1 200 OK
03:40:43.882 [Thread-1] DEBUG de.vivistra.telegrambot.client.Bot - {"ok":true,"result":[]}
03:40:43.882 [Thread-1] DEBUG de.vivistra.telegrambot.client.Bot - POST https://api.telegram.org/******/getUpdates HTTP/1.1
03:40:57.368 [Thread-1] DEBUG de.vivistra.telegrambot.client.Bot - HTTP/1.1 502 Bad Gateway
03:40:57.369 [Thread-1] DEBUG de.vivistra.telegrambot.client.Bot - <html><head><title>502 Bad Gateway</title></head><body bgcolor="white"><center><h1>502 Bad Gateway</h1></center><hr><center>nginx/1.9.1</center></body></html>
Exception in thread "Thread-1" org.json.JSONException: A JSONObject text must begin with '{' at 1 [character 2 line 1] 
        at org.json.JSONTokener.syntaxError(JSONTokener.java:433)
        at org.json.JSONObject.<init>(JSONObject.java:197)
        at de.vivistra.telegrambot.client.BotResponse.<init>(BotResponse.java:20)
        at de.vivistra.telegrambot.client.Bot.post(Bot.java:122)
        at de.vivistra.telegrambot.receiver.Receiver$ReceiverThread.run(Receiver.java:70)
```

Since the server occasionally gives me an error 502 by **HTML** format, BotResponse class throws  JSONException and stops :(

**Comment by marzn** [comment]
Thank you for this bug report. I will bring a fix the next days.

**Comment by chalkpe** [comment]
:+1:

**Comment by marzn** [comment]
It is fixed in v1.0.0. Have fun!
